language/tools/move-mv-llvm-compiler/src/stackless/translate.rs

```diff
@@ -175,11 +175,11 @@ impl<'mm, 'up> ModuleContext<'mm, 'up> {
                     }
                 };
 
-                let ll_parm_tys = fn_env
-                    .get_parameter_types()
-                    .iter()
-                    .map(|mty| self.llvm_type(mty))
-                    .collect::<Vec<_>>();
+                // stepping thru each type for better debugging
+                let _par_types = fn_env.get_parameter_types();
+                let _it = _par_types.iter();
+                let _map = _it.map(|mty| self.llvm_type(mty));
+                let ll_parm_tys = _map.collect::<Vec<_>>();
```

Comment:
You've probably discovered this, but the `_` prefix in rust is only to silence warnings about unused variables, and here several of yours are actually used.